src/polyfill/OESVertexArrayObject.js

```diff
@@ -1,5 +1,7 @@
 // https://github.com/KhronosGroup/WebGL/blob/master/sdk/demos/google/resources/OESVertexArrayObject.js
 
+import { platform } from '../core/platform.js';
```

Comment:
I'm not too keen on this. Polyfills shouldn't have any dependency on engine code. Could `setupVertexArrayObject` just be called from engine code if we're running in the browser?

Reply:
Since the engine depends on this polyfill now, we'll need it on both browser and headless. The problem is the function must be registered _somewhere_ for it to be called by the graphics device. The only alternative I can think of is converting this polyfill to a module and importing the function into graphice-device instead?

Reply:
Yeah, I think I could live with that. I mean, this doesn't feel like a true polyfill anyway. Maybe this is the different between a shim and a polyfill...

Reply:
done